### PR TITLE
chore: remove short CLI args

### PR DESCRIPTION
**Pull request by Rqnsom**
Short CLI args are not used in aptos-core, so our tooling should align. Also: use the `package-dir` (which is more common in `aptos-core`) instead of the `package-path` option.

### CODE REVIEW

File: move-mutation-test/src/cli.rs [hunk 1]
```diff
@@ -112,7 +111,7 @@ pub struct TestBuildConfig {
     ///
     /// Used mainly for disabling mutants with infinite loops.
     /// The default value is large enough for all normal tests in most projects.
-    #[clap(long = "gas-limit", default_value_t = 1_000_000)]
+    #[clap(long, default_value_t = 10_000)]
```
Comment (asmie) [inline move-mutation-test/src/cli.rs:114]: This change could be in a separate commit with some justification as to why it's 100x smaller than it used to be 😄

**Review by asmie — APPROVED** [review]
LGTM!

File: move-mutation-test/src/cli.rs (hunk 1 above)
Comment (Rqnsom) [reply move-mutation-test/src/cli.rs]: Done
